params.py

```diff
@@ -55,6 +55,15 @@
             'calibration': (EnfsiDataset(years=[2011, 2012]),),
             'test': (EnfsiDataset(years=[2011, 2012]),),
         },
+        'forenface_enfsi_sc_dev': {
+            'calibration': (ForenFaceDataset(max_num_images=10),
+                            EnfsiDataset(years=[2011]),
+                            # SCDataset(image_types=['frontal',
+                            #                        'rotated',
+                            #                        'surveillance'])
```

Comment:
klopt het dat SCData uitgecomment is?

Reply:
ja klopt, is voor de dev set. maar ik zal het nog even er helemaal uithalen